src/migrations/00014-modified-at.sql

```diff
@@ -0,0 +1,84 @@
+-- The commented sections of this file must be manually run as they affect models
+-- that are loaded prior to the migration system being available
+
+-- -- migrations.sbvr
+-- ALTER TABLE "migration"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- -- dev.sbvr
+-- ALTER TABLE "model"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- -- permissions.ts
+-- ALTER TABLE "actor"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "api key"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+-- ALTER TABLE "api key-has-permission"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+-- ALTER TABLE "api key-has-role"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "permission"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "role"
+-- ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
+
+-- ALTER TABLE "user"
```

Comment:
Why users and permissions tables cannot be handled by normal migrations? Is migrations mechanism relying on data there?

Reply:
Yeah, basically the migrations system is implemented as any other pinejs model for the most part and comes after users/permissions are set up since you require the right permissions to run queries against any standard model.  It can in theory be moved beforehand since it uses "root" permissions anyway but the manual migration will be required regardless so there's no incentive to change it until doing so can remove the need for a manual migration (changing it will make the code more complex)